app/server-utils/stampy.ts

```diff
@@ -169,7 +169,9 @@ const convertToQuestion = (title: string, v: CodaRow['values']): Question => ({
 export const loadQuestionDetail = withCache('questionDetail', async (question: string) => {
   const rows = await getCodaRows(
     QUESTION_DETAILS_TABLE,
-    question.match(/^\d+$/) ? 'UI ID' : 'Name',
+    // ids are now alphanumerical, so not possible to detect id by regex match for \d
+    // let's detect ids by length, hopefully no one will make question name so short
+    question.length <= 6 ? 'UI ID' : 'Name',
```

Comment:
perhaps `/^[\dA-Z]{3,5}$/` would be safer? Though this should be fine (unless it starts getting weird characters...)

Reply:
nah, removing the match by Name would be best, but I am not sure if any part of the UI still needs it (I would hope not, but I'm a bit tired right now to check everything - feel free to check and if not needed, make a PR to remove match by `Name` 🤞)